Silicon/NXP/iMXPlatformPkg/Drivers/SmbiosConfigDxe/SmbiosConfigDxe.c

```diff
@@ -0,0 +1,945 @@
+/** @file
+*
+*  Copyright (c) 2019 Microsoft Corporation. All rights reserved.
+*
+*  This program and the accompanying materials
+*  are licensed and made available under the terms and conditions of the BSD License
+*  which accompanies this distribution.  The full text of the license may be found at
+*  http://opensource.org/licenses/bsd-license.php
+*
+*  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+*
+**/
+
```

Comment:
Might be nice to have a "why does this exist" section here for future reference, briefly explaining the general flow of how SMBIOS override works (including Henry's provisioning flow as well maybe?) and why we did it. Or maybe in the PlatformSMBIOS as an alternative location?

Reply:
Added